frontend/components/panels/ProfitCheckPanel.vue

```diff
@@ -1,10 +1,14 @@
 <template>
     <BasePanel :current-state="currentStateAndTitle.name">
         <template #title> {{ currentStateAndTitle.title }} </template>
-        <TextBlock>
+        <TextBlock v-if="currentStateAndTitle.title === 'The transaction gross profit is negative'">
             You cannot execute an auction while it is not yet profitable. Once the auction price drops below the price
             on UniSwap, you may continue with the auction participation
         </TextBlock>
+        <TextBlock v-else>
+            Executing an auction with a negative net profit is possible, but according to our calculations it will
+            result in a net loss due to the transaction fees being higher than the gross profit.
+        </TextBlock>
```

Comment:
Currently, this text will be displayed when `The transaction profit is unknown` and `The transaction profit is positive`. I don't think it's desired. I think we should aim for the
- Generic text what explains what are both things are in one text
- "Negative net profit" text that is shown only when `notice`